classical-components-demo/ocr/src/main/java/io/scanbot/example/MainActivity.kt

```diff
@@ -134,40 +145,45 @@ class MainActivity : AppCompatActivity() {
     This AsyncTask is used here only for the sake of example. Please, try to avoid usage of
```

Comment:
Remove comment - not applicable anymore. Remove here and in other modules where asynctask is no more present